logging/bin/deploy_esexporter.sh

```diff
@@ -31,7 +33,7 @@ fi
 # get credentials
 get_credentials_from_secret metricgetter
 rc=$?
-if [ "$rc" != "0" ] ;then log_info "RC=$rc"; exit $rc;fi
+if [ "$rc" != "0" ] ;then log_verbose "RC=$rc"; exit $rc;fi
 
```

Comment:
I think can/should be log_debug.  Not sure why it was ever log_info.

Reply:
I have gone ahead and updated it in the files that I've touched.
Do you want me to go ahead and update it to DEBUG in **deploy_elasticsearch_content_open.sh** and **deploy_kibana_content.sh** files?

Reply:
I believe I've already done that as part of my current Pull Request; so I think we are good.